Extension/MultipathUpgrade/Enum/EquipmentProperties.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace YAPO.MultipathUpgrade.Enum
+{
+    [Flags]
+    public enum EquipmentProperties
+    {
+        NONE = 0,
+        HAS_SHIELD = 1,
+        HAS_CROSS_BOW = 2,
+        HAS_BOW = 4
```

Comment:
It's a Flags enum. Multiple flags can be set bitwise ex:

Bowmen generally have bows so it would be 0001 which is 4
Crossbowman has a shield and a crossbow that means value would be 0110, which is 3

So if HAS_BOW has value 3 the code cant differ between crossbowmen with shields and bowmen